services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowConnectedDeviceSpec.groovy

```diff
@@ -467,18 +552,21 @@ srcLldpDevices=#newSrcEnabled, dstLldpDevices=#newDstEnabled"() {
                 it.size() == 1
                 it[0].portNumber == tg.switchPort
                 it[0].lldp.first().vlan == flow.source.vlanId
-                it[0].lldp.first().flowId == null //questionable. May want to link this to a flow
+                it[0].lldp.first().flowId == flow.id
                 verifyEquals(it[0].lldp.first(), lldpData)
             }
         }
 
-        then: "Device is NOT registered as a flow device"
-        northbound.getFlowConnectedDevices(flow.id).source.lldp.empty
+        then: "Device is registered as a flow device"
+        !northbound.getFlowConnectedDevices(flow.id).source.lldp.empty
```

Comment:
why? Feature is off on flow level. Why it is still being registered per-flow?

Reply:
Well you know, that might make sense actually..need to get a common understanding on this one